src/additional-scss/_global-styles.scss

```diff
@@ -35,9 +35,9 @@ body {
   line-height: 2.5em;
   text-align: center;
   border-radius: 50%;
-  background: $epa-info-base;
+  background: white;
```

Comment:
nitpick: could we please chance to $epa-white?  it's a scss variable in _variables.scss

Reply:
Feedback was addressed with most recent push. Pull request has been approved